text/0000-wheel-platform-support.md

```diff
@@ -0,0 +1,217 @@
+- Feature Name: "Wheel" support in RubyGems
+- Start Date: 2025-08-20
+- RFC PR: (leave this empty)
+- Bundler Issue: (leave this empty)
+
+# Summary
+
+Add wheel-format platform support to RubyGems for more fine-grained matching for binary gem distribution through ruby/api tags and platform tags (`whl-{abi_tags}-{platform_tags}`). This enables "platform matching" based on Ruby ABI version, os, os version, cpu architecture, and libc implementation (for generic linux platforms).
```

Comment:
Is it possible (in theory) to make wheel with only ruby code inside eligible for all platforms?